src/convert/metadataConverter.ts

```diff
@@ -111,7 +117,10 @@ export class MetadataConverter {
       }
 
       const conversionPipeline = pipeline(
-        new ComponentReader(components),
+        Readable.from(components),
+        !targetFormatIsSource && (process.env.DEBUG_REPLACEMENTS_VIA_CONVERT === 'true' || output.type === 'zip')
```

Comment:
want to keep this env var in?

Reply:
I'm leaning toward yes.  It would also enable a use case I don't want to officially encourage (do replacements to create a mdapi folder that you can mdapi deploy)

Reply:
but I renamed it to match `sf` style